test.webpack.config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
```

Comment:
See if we can put this in the config folder as well.
It's defined in babelrc if I recall correctly.